demisto_sdk/commands/update_release_notes/update_rn.py

```diff
@@ -257,76 +261,77 @@ def build_rn_template(self, changed_items: dict):
         widgets_header = False
         dashboards_header = False
         connections_header = False
-        for k, v in sorted(changed_items.items(), key=lambda x: x[1]['type'] if x[1] is not None else ''):
-            desc = v.get('description', '')
-            is_new_file = v.get('is_new_file', False)
-            v = v.get('type', '')
-            if not v:
+        for content_name, data in sorted(changed_items.items(), key=lambda x: x[1]['type'] if x[1] is not None else ''):
+            desc = data.get('description', '')
+            is_new_file = data.get('is_new_file', False)
+            _type = data.get('type', '')
+            if not _type:
                 continue
 
-            if v in ('Connections', 'Incident Types', 'Indicator Types', 'Layouts', 'Incident Fields'):
-                rn_desc = f'- **{k}**\n'
+            if _type in ('Connections', 'Incident Types', 'Indicator Types', 'Layouts', 'Incident Fields'):
+                rn_desc = f'- **{content_name}**\n'
             else:
-                rn_desc = f'##### {k}\n- {desc}\n' if is_new_file else f'##### {k}\n- %%UPDATE_RN%%\n'
+                rn_desc = f'##### {content_name}\n- {desc}\n' if is_new_file \
```

Comment:
missed that one the first time...
```suggestion
                rn_desc = f'##### New: {content_name}\n- {desc}\n' if is_new_file \
```